packages/block-library/src/cover/edit.js

```diff
@@ -187,33 +191,52 @@ class CoverEdit extends Component {
 						</>
 					) }
 				</BlockControls>
-				{ !! url && (
-					<InspectorControls>
-						<PanelBody title={ __( 'Cover Settings' ) }>
-							{ IMAGE_BACKGROUND_TYPE === backgroundType && (
+				<InspectorControls>
+					{ !! url && (
+						<PanelBody title={ __( 'Media Settings' ) }>
+							{ !! url && ( IMAGE_BACKGROUND_TYPE === backgroundType ) && (
```

Comment:
Is there any case where IMAGE_BACKGROUND_TYPE === backgroundType and no URL exists? What is the reason this URL check is being added? 
We are doing the URL check in line 195 so I think we don't need to repeat it here.

Reply:
👍🏻